SECURITY.md

```diff
@@ -0,0 +1,90 @@
+# Security Process and Policy
+
+This document provides the details on the veraison/go-cose security policy and details the processes surrounding security handling.
+
+## Supported Versions
+
+[go-cose][go-cose] is currently is in active development, moving to a [1.0.0 release][v1.0.0-milestone]. The latest pre-release will be supported until 1.0.0 is released. As 1.0.0 is released, pre-release references will need to be redirected 1.0.0.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| [v1.0.0-alpha1][v1.0.0-alpha1-release]  | :green_check_mark: |
+
+## Report A Vulnerability
+
+We’re extremely grateful for security researchers and users who report vulnerabilities
+to the [veraison/go-cose][go-cose] community. All reports are thoroughly investigated by a set of [veraison/go-cose maintainers][go-cose-maintainers].
+
+To make a report please email the private security list at <a href="mailto:test@example.com?subject=go-cose Security Notification">go-cose-security@googlegroups.com</a> with details using the following template:
+
+### Reporting Template
+
+```console
+[TO:]:     go-cose-security@googlegroups.com
+[SUBJECT]: go-cose Security Notification
+[BODY]:
+Release: v1.0.0-alpha.1
+
+Summary:
+A quick summary of the issue
+
+Impact:
+Details on how to reproduce the security issue.
+
+Contact:
+Information on who to contact for additional information
+```
+
+### When To Send a Report
+
+You think you have found a vulnerability in th [veraison/go-cose][go-cose] project or a dependency of a Veraison project. This can be any of the repositories on the [Veraison GitHub organization](https://github.com/veraison).
```

Comment:
A small typo:

```suggestion
You think you have found a vulnerability in the [veraison/go-cose][go-cose] project or a dependency of a Veraison project. This can be any of the repositories on the [Veraison GitHub organization](https://github.com/veraison).
```

And a slightly more substantive comment:

Why do we need to cover any Veraison project here?  I guess the scope should be go-cose and its deps only.

Reply:
Reduced scope.